packages/web-components/src/components/rux-input/rux-input.scss

```diff
@@ -62,57 +64,59 @@
         position: relative;
         box-sizing: border-box;
         order: 2;
-        border: 1px solid var(--color-border-interactive-muted);
+        // use box shadow instead of border to create borders
+        box-shadow: var(--color-border-interactive-muted) 0 0 0 1px inset;
         border-radius: var(--radius-base);
         font-family: var(--font-body-1-font-family);
         font-size: var(--font-body-1-font-size);
         font-weight: var(--font-body-1-font-weight);
+        //control-body line heights not yet implemented
+        line-height: var(--line-height-sm);
         letter-spacing: var(--font-body-1-letter-spacing);
         color: var(--color-text-primary);
         background-color: var(--color-background-base-default);
 
         &--invalid {
-            border: 1px solid var(--color-text-error);
+            box-shadow: var(--color-border-error) 0 0 0 1px inset;
         }
         &--disabled {
             opacity: 0.4;
             opacity: var(--disabled-opacity);
             cursor: not-allowed;
         }
         &--small {
-            padding: 0.219rem 0.5rem; // 3.5px 8px;
-            min-height: 1.625rem; //26px
+            padding: var(--spacing-1) var(--spacing-2); // 4px 8px
         }
         &--medium {
-            padding: 0.469rem 0.5rem; // 7.5px 8px;
-            min-height: 2.25rem; //36px
+            padding: var(--spacing-2); //8px
         }
         &--large {
-            padding: 0.844rem 0.5rem; //13.5px 8px;
-            min-height: 2.875rem; //46px
+            padding: var(--spacing-3) var(--spacing-2); //12px 8px
         }
         &--search {
             -webkit-appearance: none;
             -moz-appearance: none;
-            padding-left: 2.25rem; //36px
+            padding-left: calc(var(--spacing-8)); //32px
```

Comment:
is this calc doing anything?

Reply:
Nope! It was but then I was able to remove the calculation and then my brain oozed out of my ears. Fixed!